src/prefab/install.ts

```diff
@@ -12,13 +12,27 @@ export default async function install(pkg: Package, logger?: Logger): Promise<In
 
   const cellar = useCellar()
   const tea_prefix = usePrefix()
-  const { isCI, dryrun, json } = useConfig()
+  const { isCI, dryrun, json, env } = useConfig()
   const compression = get_compression(isCI)
   const stowage = StowageNativeBottle({ pkg: { project, version }, compression })
   const url = useOffLicense('s3').url(stowage)
   const tarball = useCache().path(stowage)
   const shelf = tea_prefix.join(pkg.project)
 
+  if(env.TEA_MAGIC === "prompt") {
+    do {
+      const val = prompt(`┌ ⚠️  Tea requests to install ${pkg.project} (v${pkg.version})\n└ \x1B[1mAllow?\x1B[0m [y/n]`)?.toLowerCase();
+      // If val is undefined, there was no prompt given since this isn't an interactive tty
+      if(!val || val === "y") {
+        break;
+      }
+      if(val === "n") {
+        Deno.exit(1);
```

Comment:
@ABevier are we allowed to call exit in the codebase anymore? Or we have to throw an ExitError?

Reply:
@mxcl Ideally we'd throw an exit error.  Otherwise we can't test this path because it ends the unit test process.

Reply:
@ABevier could you point me to such an "Exit Error" example?

Reply:
https://github.com/teaxyz/cli/blob/4b2c03abc6fb7c177be6ef73e05ae73eeac6aa58/src/app.exec.ts#L30